src/main/java/Ponder_Pika/Main.java

```diff
@@ -0,0 +1,8 @@
+package Ponder_Pika;
+
+public class Main {
+    public static void main(String[] args) {
+        Ponder_Pika bot = new Ponder_Pika();
```

Comment:
Is Ponder_Pika in PascalCase?

Reply:
It is not, I have changed it completely. I did my check style after your reviews were given.